bouncerscript/src/bouncerscript/constants.py

```diff
@@ -70,39 +70,39 @@
 
 _BOUNCER_PATH_REGEXES_PER_PRODUCT_DEFAULT = {
     "firefox-nightly-latest": (
-        r"^(/firefox/nightly/latest-mozilla-central-l10n/firefox-\d+\.0a1\.:lang\."
+        r"^(/firefox/nightly/latest-(mozilla-central|try)-l10n/firefox-\d+\.0a1\.:lang\."
```

Comment:
This is not ideal, but given that all we do is version bump these, it's probably OK? I could be convinced to move it to the config...